tests/algorithms/test_bounded_mean.py

```diff
@@ -1,10 +1,10 @@
 import os
 import pytest
 from pydp.algorithms.laplacian import BoundedMean
-from pydp._pydp import Summary
+from pydp._pydp import Summary  # type: ignore
 
 
-expect_near = lambda expected, actual, tol: (
+def expect_near(expected, actual, tol): return (
```

Comment:
is there a reason to move from lambda to this new implementation?

Reply:
It's not what I intended, and I reset the commit